### ISSUE TITLE
Resolution very low

### ISSUE THREAD

**Issue by mfouesneau** [brendena/MathEquationsGoogleSlide#14]
The rendered equations have low resolution.

**Comment by LoganDark** [comment]
![image](https://user-images.githubusercontent.com/4723091/49956681-a9be6700-feba-11e8-8e9d-c906f1704a36.png)
Make the number bigger.

**Comment by mfouesneau** [comment]
Does not make the resolution bigger. Only the image itself.

**Comment by LoganDark** [comment]
Sorry then, I don't know how to help. Thought that controlled font size like an actually _smart_ program would, I guess not...

**Comment by vittau** [comment]
Agreed, can you please increase the rendering resolution for the equations? Wide equations become very low res.
I have a feeling it's line 297 in index.html which causes MathJax to render at that width. Might do a pull request later, but I suppose it's a very easy fix.

**Comment by oesst** [comment]
Still the same problem.
